integration_test/general.py

```diff
@@ -0,0 +1,149 @@
+import os
+import random
+import string
+
+ERROR_CODE = 1
+
+def runCommand(command, message):
+    result = os.system(command)
+    print(message, result)
+    return result
+
+def get_random_string(length):
+    letters = string.ascii_lowercase
+    result_str = ''.join(random.choice(letters) for i in range(length))
+    print("Store name is:", result_str)
+    return result_str
+
+# get current host from env
+def get_host():
+   return os.environ.get("HOST")
+
+# clear data
+def clear_data():
+    return runCommand("rm -R dcm4che", "remove dcm4che")
+
+# install environment
+def install_environment():
+    return runCommand("./integration_test/scripts/install-env.sh", "install environment exit with")
+
+# clone-dcm4che
+def clone_dcm4che():
+    return runCommand("git clone https://github.com/dcm4che/dcm4che.git dcm4che", "clone-dcm4che exit with")
+
+# checkout-dcm4che-tag
+def checkout_dcm4che_tag():
+    return runCommand("cd dcm4che && git checkout \"tags/5.15.1\"", "checkout-dcm4che-tag exit with")
+
+# build-tools
+def build_tools():
+    return runCommand("./integration_test/scripts/mvn-install-tools.sh", "build-tools exit with")
+
+# store-scp
+def store_scp(store_csp_run_step, store_scp_port, close_store_scp_run_step):
+    return runCommand("./integration_test/scripts/run-store-scp-and-wait.sh " + store_csp_run_step + " "+store_scp_port+" "+close_store_scp_run_step, "store-scp exit with")
+
+# build adapter
+def build_adapter():
+    return runCommand("./integration_test/scripts/build-adapters.sh", "build adapter exit with")
+
+# build adapter image
+def build_adapter_image(imageproject):
+    return runCommand("./integration_test/scripts/build-adapter-image.sh "+imageproject+" local_run", "build adapter image exit with")
+
+# setup-dataset-and-dicom-store
+def setup_dataset_and_dicom_store(project, location, dataset, store_name):
+    return runCommand("./integration_test/scripts/setup-dataset-and-store.sh "+project+" "+location+" "+dataset+" "+store_name, "setup-dataset-and-dicom-store exit with")
+
+# run import adapter
+def run_import_adapter(adapter_port, version, project, location, dataset, store_name, store_scp_run_step, store_scp_port, com_scu_step, com_scu_port, imageproject, sha):
+    return runCommand("./integration_test/scripts/run-import-adapter.sh "+adapter_port+" "+version+" "+project+" "+location+" "+dataset+" "+store_name+" "+store_scp_run_step+" "+store_scp_port+" "+com_scu_step+" "+com_scu_port+" "+imageproject+" "+sha+" &", "run import adapter exit with")
```

Comment:
I'm confused why imageproject, sha, and & are passed to the script, it does not seem to use them? I believe they could be deleted.